sympy/solvers/ode/tests/test_ode.py

```diff
@@ -1094,3 +1094,12 @@ def test_issue_23425():
     assert classify_ode(eq) == \
         ('Liouville', 'nth_order_reducible', \
         '2nd_power_series_ordinary', 'Liouville_Integral')
+
+
+def test_issue_25820():
```

Comment:
I thought that must faster test went through the added code-path but will have to recheck since this does not appear to fail in master (but maybe did so for some other reason).